test/e2e/data/infrastructure-docker/v1.4/bases/cluster-with-topology.yaml

```diff
@@ -18,9 +18,11 @@ spec:
     controlPlane:
       metadata:
         labels:
```

Comment:
These files should be created based on the files in main.

Basically the idea is to keep the files for old providers as close to main as we can.

We only should have differences if e.g. we can't set a field because it was only introduced in main.

I synced the v1.4 folder with the main folder and adjusted it accordingly.